config/development.json5

```diff
@@ -1,10 +1,14 @@
 {
   blockchain: {
+    contractAddress: '0x5Ebbf329a6AACEdC2C78A6424E02524c436B8825',
```

Comment:
Hmm why do you add the contract address here? Is it always the same? I have for example different address...

Reply:
oops. sorry